2025-08-17 01:02:05,956 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:02:05,957 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:02:05,958 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:02:05,959 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=9420 <<<
2025-08-17 01:02:13,567 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:02:13,598 - RouterManager - ERROR - Error processing socket message: 'VirtualNetwork' object has no attribute 'forward_file'
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 89, in _process_socket_message
    self.network.forward_file(None, node_name)
AttributeError: 'VirtualNetwork' object has no attribute 'forward_file'
2025-08-17 01:02:20,862 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:02:20,907 - RouterManager - ERROR - Error processing socket message: 'VirtualNetwork' object has no attribute 'forward_file'
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 89, in _process_socket_message
    self.network.forward_file(None, node_name)
AttributeError: 'VirtualNetwork' object has no attribute 'forward_file'
2025-08-17 01:08:03,753 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=9420 <<<
2025-08-17 01:08:03,753 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:08:03,754 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:08:03,755 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:08:33,640 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:08:33,640 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:08:33,641 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:08:33,669 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=8764 <<<
2025-08-17 01:08:36,290 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:08:39,502 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:08:56,013 - pyftpdlib - INFO - 127.0.0.1:64243-[] FTP session opened (connect)
2025-08-17 01:08:56,151 - pyftpdlib - INFO - 127.0.0.1:64243-[user] USER 'user' logged in.
2025-08-17 01:08:59,386 - pyftpdlib - INFO - 127.0.0.1:64243-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.813
2025-08-17 01:08:59,634 - RouterManager - INFO - Received chunk 1/5 for 20Mb (folder: file_001, target: node2)
2025-08-17 01:08:59,696 - pyftpdlib - INFO - 127.0.0.1:64243-[user] FTP session closed (disconnect).
2025-08-17 01:09:11,568 - pyftpdlib - INFO - 127.0.0.1:64246-[] FTP session opened (connect)
2025-08-17 01:09:11,643 - pyftpdlib - INFO - 127.0.0.1:64246-[user] USER 'user' logged in.
2025-08-17 01:09:14,663 - pyftpdlib - INFO - 127.0.0.1:64246-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.625
2025-08-17 01:09:14,944 - RouterManager - INFO - Received chunk 1/5 for 20Mb (folder: file_002, target: node2)
2025-08-17 01:09:15,006 - pyftpdlib - INFO - 127.0.0.1:64246-[user] FTP session closed (disconnect).
2025-08-17 01:09:36,265 - pyftpdlib - INFO - 127.0.0.1:64250-[] FTP session opened (connect)
2025-08-17 01:09:36,356 - pyftpdlib - INFO - 127.0.0.1:64250-[user] USER 'user' logged in.
2025-08-17 01:09:38,223 - pyftpdlib - INFO - 127.0.0.1:64250-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.453
2025-08-17 01:09:38,531 - RouterManager - INFO - Received chunk 1/5 for 10Mb (folder: file_003, target: node2)
2025-08-17 01:09:38,626 - pyftpdlib - INFO - 127.0.0.1:64250-[user] FTP session closed (disconnect).
2025-08-17 01:11:02,973 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=8764 <<<
2025-08-17 01:11:02,973 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:11:02,974 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:11:02,975 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:11:09,366 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:11:09,366 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:11:09,367 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:11:09,368 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=7304 <<<
2025-08-17 01:11:11,900 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:11:14,924 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:11:25,313 - pyftpdlib - INFO - 127.0.0.1:64275-[] FTP session opened (connect)
2025-08-17 01:11:25,423 - pyftpdlib - INFO - 127.0.0.1:64275-[user] USER 'user' logged in.
2025-08-17 01:11:25,933 - pyftpdlib - INFO - 127.0.0.1:64275-[user] FTP session closed (disconnect).
2025-08-17 01:11:26,011 - pyftpdlib - INFO - 127.0.0.1:64278-[] FTP session opened (connect)
2025-08-17 01:11:26,121 - pyftpdlib - INFO - 127.0.0.1:64278-[user] USER 'user' logged in.
2025-08-17 01:11:28,019 - pyftpdlib - INFO - 127.0.0.1:64278-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.438
2025-08-17 01:11:28,314 - RouterManager - INFO - Received chunk 1/5 for 10Mb (folder: file_004, target: node2)
2025-08-17 01:11:30,184 - pyftpdlib - INFO - 127.0.0.1:64278-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.485
2025-08-17 01:11:30,385 - RouterManager - INFO - Received chunk 2/5 for 10Mb (folder: file_004, target: node2)
2025-08-17 01:11:32,315 - pyftpdlib - INFO - 127.0.0.1:64278-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.438
2025-08-17 01:11:32,520 - RouterManager - INFO - Received chunk 3/5 for 10Mb (folder: file_004, target: node2)
2025-08-17 01:11:33,844 - pyftpdlib - INFO - 127.0.0.1:64278-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=0.938
2025-08-17 01:11:33,993 - RouterManager - INFO - Received chunk 4/5 for 10Mb (folder: file_004, target: node2)
2025-08-17 01:11:35,953 - pyftpdlib - INFO - 127.0.0.1:64278-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.531
2025-08-17 01:11:36,185 - RouterManager - INFO - Received chunk 5/5 for 10Mb (folder: file_004, target: node2)
2025-08-17 01:11:36,232 - RouterManager - INFO - Stored 10Mb in folder file_004 for node2
2025-08-17 01:11:36,263 - RouterManager - INFO - Target node node2 is active, forwarding file 10Mb
2025-08-17 01:11:36,400 - pyftpdlib - INFO - 127.0.0.1:64278-[user] FTP session closed (disconnect).
2025-08-17 01:13:05,395 - pyftpdlib - INFO - 127.0.0.1:64317-[] FTP session opened (connect)
2025-08-17 01:13:05,503 - pyftpdlib - INFO - 127.0.0.1:64317-[user] USER 'user' logged in.
2025-08-17 01:13:06,078 - pyftpdlib - INFO - 127.0.0.1:64317-[user] FTP session closed (disconnect).
2025-08-17 01:13:06,126 - pyftpdlib - INFO - 127.0.0.1:64320-[] FTP session opened (connect)
2025-08-17 01:13:06,251 - pyftpdlib - INFO - 127.0.0.1:64320-[user] USER 'user' logged in.
2025-08-17 01:13:09,360 - pyftpdlib - INFO - 127.0.0.1:64320-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.718
2025-08-17 01:13:09,400 - RouterManager - INFO - Received chunk 1/5 for 20Mb (folder: file_004, target: node2)
2025-08-17 01:13:11,825 - pyftpdlib - INFO - 127.0.0.1:64320-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.375
2025-08-17 01:13:11,995 - RouterManager - INFO - Received chunk 2/5 for 20Mb (folder: file_004, target: node2)
2025-08-17 01:13:15,300 - pyftpdlib - INFO - 127.0.0.1:64320-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.891
2025-08-17 01:13:15,517 - RouterManager - INFO - Received chunk 3/5 for 20Mb (folder: file_004, target: node2)
2025-08-17 01:13:18,830 - pyftpdlib - INFO - 127.0.0.1:64320-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.812
2025-08-17 01:13:19,017 - RouterManager - INFO - Received chunk 4/5 for 20Mb (folder: file_004, target: node2)
2025-08-17 01:13:22,256 - pyftpdlib - INFO - 127.0.0.1:64320-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.812
2025-08-17 01:13:22,460 - RouterManager - INFO - Received chunk 5/5 for 20Mb (folder: file_004, target: node2)
2025-08-17 01:13:22,492 - RouterManager - INFO - Stored 20Mb in folder file_004 for node2
2025-08-17 01:13:22,523 - RouterManager - INFO - Target node node2 is active, forwarding file 20Mb
2025-08-17 01:13:22,726 - pyftpdlib - INFO - 127.0.0.1:64320-[user] FTP session closed (disconnect).
2025-08-17 01:18:29,781 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=7304 <<<
2025-08-17 01:18:29,781 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:18:29,783 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:18:29,783 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:18:32,112 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:18:32,113 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:18:32,113 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:18:32,140 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=19784 <<<
2025-08-17 01:18:38,467 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:19:10,660 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:24:18,819 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:24:18,819 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=19784 <<<
2025-08-17 01:24:18,820 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:24:18,820 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:24:37,331 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:24:37,332 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:24:37,332 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:24:37,333 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=20568 <<<
2025-08-17 01:24:40,051 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:24:43,646 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:24:57,238 - pyftpdlib - INFO - 127.0.0.1:64481-[] FTP session opened (connect)
2025-08-17 01:24:57,346 - pyftpdlib - INFO - 127.0.0.1:64481-[user] USER 'user' logged in.
2025-08-17 01:24:57,894 - pyftpdlib - INFO - 127.0.0.1:64481-[user] FTP session closed (disconnect).
2025-08-17 01:24:57,956 - pyftpdlib - INFO - 127.0.0.1:64484-[] FTP session opened (connect)
2025-08-17 01:24:58,064 - pyftpdlib - INFO - 127.0.0.1:64484-[user] USER 'user' logged in.
2025-08-17 01:25:00,741 - pyftpdlib - INFO - 127.0.0.1:64484-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.187
2025-08-17 01:25:01,115 - RouterManager - INFO - Received chunk 1/5 for 20Mb (folder: file_005, target: node2)
2025-08-17 01:25:01,432 - pyftpdlib - INFO - 127.0.0.1:64484-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=0.219
2025-08-17 01:25:01,630 - RouterManager - INFO - Received chunk 2/5 for 20Mb (folder: file_005, target: node2)
2025-08-17 01:25:04,728 - pyftpdlib - INFO - 127.0.0.1:64484-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.609
2025-08-17 01:25:04,929 - RouterManager - INFO - Received chunk 3/5 for 20Mb (folder: file_005, target: node2)
2025-08-17 01:25:08,155 - pyftpdlib - INFO - 127.0.0.1:64484-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.812
2025-08-17 01:25:08,355 - RouterManager - INFO - Received chunk 4/5 for 20Mb (folder: file_005, target: node2)
2025-08-17 01:25:11,409 - pyftpdlib - INFO - 127.0.0.1:64484-[user] STOR E:\cloud-architecture-simulation\assets\server\20Mb completed=1 bytes=4194336 seconds=2.609
2025-08-17 01:25:11,580 - RouterManager - INFO - Received chunk 5/5 for 20Mb (folder: file_005, target: node2)
2025-08-17 01:25:11,626 - RouterManager - INFO - Stored 20Mb in folder file_005 for node2
2025-08-17 01:25:11,673 - RouterManager - INFO - Target node node2 is active, forwarding file 20Mb
2025-08-17 01:25:11,768 - pyftpdlib - INFO - 127.0.0.1:64484-[user] FTP session closed (disconnect).
2025-08-17 01:27:58,406 - pyftpdlib - INFO - 127.0.0.1:64544-[] FTP session opened (connect)
2025-08-17 01:27:58,531 - pyftpdlib - INFO - 127.0.0.1:64544-[user] USER 'user' logged in.
2025-08-17 01:27:59,016 - pyftpdlib - INFO - 127.0.0.1:64544-[user] FTP session closed (disconnect).
2025-08-17 01:27:59,071 - pyftpdlib - INFO - 127.0.0.1:64547-[] FTP session opened (connect)
2025-08-17 01:27:59,155 - pyftpdlib - INFO - 127.0.0.1:64547-[user] USER 'user' logged in.
2025-08-17 01:28:00,693 - pyftpdlib - INFO - 127.0.0.1:64547-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.516
2025-08-17 01:28:01,037 - RouterManager - INFO - Received chunk 1/5 for 10Mb (folder: file_005, target: node2)
2025-08-17 01:28:02,700 - pyftpdlib - INFO - 127.0.0.1:64547-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.297
2025-08-17 01:28:02,918 - RouterManager - INFO - Received chunk 2/5 for 10Mb (folder: file_005, target: node2)
2025-08-17 01:28:04,785 - pyftpdlib - INFO - 127.0.0.1:64547-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.39
2025-08-17 01:28:04,989 - RouterManager - INFO - Received chunk 3/5 for 10Mb (folder: file_005, target: node2)
2025-08-17 01:28:06,951 - pyftpdlib - INFO - 127.0.0.1:64547-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.531
2025-08-17 01:28:07,089 - RouterManager - INFO - Received chunk 4/5 for 10Mb (folder: file_005, target: node2)
2025-08-17 01:28:08,973 - pyftpdlib - INFO - 127.0.0.1:64547-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.469
2025-08-17 01:28:09,193 - RouterManager - INFO - Received chunk 5/5 for 10Mb (folder: file_005, target: node2)
2025-08-17 01:28:09,223 - RouterManager - INFO - Stored 10Mb in folder file_005 for node2
2025-08-17 01:28:09,268 - RouterManager - INFO - Target node node2 is active, forwarding file 10Mb
2025-08-17 01:28:09,409 - pyftpdlib - INFO - 127.0.0.1:64547-[user] FTP session closed (disconnect).
2025-08-17 01:28:22,850 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:29:25,943 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=20568 <<<
2025-08-17 01:29:25,943 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:29:25,944 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:29:25,945 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:29:29,458 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:29:29,459 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:29:29,472 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:29:29,534 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=17616 <<<
2025-08-17 01:30:22,226 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:30:36,145 - pyftpdlib - INFO - 127.0.0.1:64594-[] FTP session opened (connect)
2025-08-17 01:30:36,255 - pyftpdlib - INFO - 127.0.0.1:64594-[user] USER 'user' logged in.
2025-08-17 01:30:36,705 - pyftpdlib - INFO - 127.0.0.1:64594-[user] FTP session closed (disconnect).
2025-08-17 01:30:36,753 - pyftpdlib - INFO - 127.0.0.1:64597-[] FTP session opened (connect)
2025-08-17 01:30:36,862 - pyftpdlib - INFO - 127.0.0.1:64597-[user] USER 'user' logged in.
2025-08-17 01:30:38,622 - pyftpdlib - INFO - 127.0.0.1:64597-[user] STOR E:\cloud-architecture-simulation\assets\server\10Mb completed=1 bytes=2097184 seconds=1.391
2025-08-17 01:30:38,747 - RouterManager - WARNING - Target node node3 is not active, rejecting file transfer
2025-08-17 01:30:38,826 - pyftpdlib - INFO - 127.0.0.1:64597-[user] FTP session closed (disconnect).
2025-08-17 01:31:24,598 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:31:24,599 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=17616 <<<
2025-08-17 01:31:24,599 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:31:24,600 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:33:55,509 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:33:55,510 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:33:55,510 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:33:55,511 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=1784 <<<
2025-08-17 01:35:17,852 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:35:36,917 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:36:04,630 - pyftpdlib - INFO - 127.0.0.1:64670-[] FTP session opened (connect)
2025-08-17 01:36:04,772 - pyftpdlib - INFO - 127.0.0.1:64670-[user] USER 'user' logged in.
2025-08-17 01:36:05,222 - pyftpdlib - INFO - 127.0.0.1:64670-[user] FTP session closed (disconnect).
2025-08-17 01:36:05,285 - pyftpdlib - INFO - 127.0.0.1:64673-[] FTP session opened (connect)
2025-08-17 01:36:05,378 - pyftpdlib - INFO - 127.0.0.1:64673-[user] USER 'user' logged in.
2025-08-17 01:36:05,890 - pyftpdlib - INFO - 127.0.0.1:64673-[user] STOR E:\cloud-architecture-simulation\assets\server\200Mb completed=1 bytes=232 seconds=0.063
2025-08-17 01:36:06,139 - RouterManager - INFO - Received chunk 1/1 for 200Mb (folder: file_001, target: node2)
2025-08-17 01:36:06,187 - RouterManager - INFO - Stored 200Mb in folder file_001 for node2
2025-08-17 01:36:06,233 - RouterManager - INFO - Target node node2 is active, forwarding file 200Mb
2025-08-17 01:36:06,358 - pyftpdlib - INFO - 127.0.0.1:64673-[user] FTP session closed (disconnect).
2025-08-17 01:37:25,218 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=1784 <<<
2025-08-17 01:37:25,219 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:37:25,219 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:37:25,220 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:37:28,563 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:37:28,565 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:37:28,565 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:37:28,566 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=13464 <<<
2025-08-17 01:37:36,105 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:37:39,150 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:38:01,805 - pyftpdlib - INFO - 127.0.0.1:64732-[] FTP session opened (connect)
2025-08-17 01:38:01,853 - pyftpdlib - INFO - 127.0.0.1:64732-[user] USER 'user' logged in.
2025-08-17 01:38:02,101 - pyftpdlib - INFO - 127.0.0.1:64732-[user] FTP session closed (disconnect).
2025-08-17 01:38:02,166 - pyftpdlib - INFO - 127.0.0.1:64735-[] FTP session opened (connect)
2025-08-17 01:38:02,304 - pyftpdlib - INFO - 127.0.0.1:64735-[user] USER 'user' logged in.
2025-08-17 01:38:02,815 - pyftpdlib - INFO - 127.0.0.1:64735-[user] STOR E:\cloud-architecture-simulation\assets\server\200Mb completed=1 bytes=232 seconds=0.079
2025-08-17 01:38:03,098 - RouterManager - INFO - Received chunk 1/1 for 200Mb (folder: file_001, target: node2)
2025-08-17 01:38:03,129 - RouterManager - INFO - Stored 200Mb in folder file_001 for node2
2025-08-17 01:38:03,161 - RouterManager - INFO - Target node node2 is active, forwarding file 200Mb
2025-08-17 01:38:03,350 - pyftpdlib - INFO - 127.0.0.1:64735-[user] FTP session closed (disconnect).
2025-08-17 01:43:10,115 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=13464 <<<
2025-08-17 01:43:10,116 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:43:10,116 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:43:10,117 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:44:08,601 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:44:08,601 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:44:08,602 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:44:08,602 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=22408 <<<
2025-08-17 01:44:12,199 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:44:15,127 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:44:34,010 - pyftpdlib - INFO - 127.0.0.1:64838-[] FTP session opened (connect)
2025-08-17 01:44:34,103 - pyftpdlib - INFO - 127.0.0.1:64838-[user] USER 'user' logged in.
2025-08-17 01:44:34,493 - pyftpdlib - INFO - 127.0.0.1:64838-[user] FTP session closed (disconnect).
2025-08-17 01:44:34,556 - pyftpdlib - INFO - 127.0.0.1:64841-[] FTP session opened (connect)
2025-08-17 01:44:34,650 - pyftpdlib - INFO - 127.0.0.1:64841-[user] USER 'user' logged in.
2025-08-17 01:45:01,496 - pyftpdlib - INFO - 127.0.0.1:64841-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=52428832 seconds=26.453
2025-08-17 01:45:01,929 - RouterManager - INFO - Received chunk 1/4 for 200M (folder: file_001, target: node2)
2025-08-17 01:45:30,741 - pyftpdlib - INFO - 127.0.0.1:64841-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=52428832 seconds=27.89
2025-08-17 01:45:31,006 - RouterManager - INFO - Received chunk 2/4 for 200M (folder: file_001, target: node2)
2025-08-17 01:45:58,999 - pyftpdlib - INFO - 127.0.0.1:64841-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=52428832 seconds=26.906
2025-08-17 01:45:59,249 - RouterManager - INFO - Received chunk 3/4 for 200M (folder: file_001, target: node2)
2025-08-17 01:46:26,038 - pyftpdlib - INFO - 127.0.0.1:64841-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=52428832 seconds=25.718
2025-08-17 01:46:26,286 - RouterManager - INFO - Received chunk 4/4 for 200M (folder: file_001, target: node2)
2025-08-17 01:46:26,316 - RouterManager - INFO - Stored 200M in folder file_001 for node2
2025-08-17 01:46:26,363 - RouterManager - INFO - Target node node2 is active, forwarding file 200M
2025-08-17 01:46:26,954 - pyftpdlib - INFO - 127.0.0.1:64841-[user] FTP session closed (disconnect).
2025-08-17 01:48:38,333 - pyftpdlib - INFO - 127.0.0.1:65006-[] FTP session opened (connect)
2025-08-17 01:48:38,396 - pyftpdlib - INFO - 127.0.0.1:65006-[user] USER 'user' logged in.
2025-08-17 01:48:38,909 - pyftpdlib - INFO - 127.0.0.1:65006-[user] FTP session closed (disconnect).
2025-08-17 01:48:38,925 - pyftpdlib - INFO - 127.0.0.1:65009-[] FTP session opened (connect)
2025-08-17 01:48:38,972 - pyftpdlib - INFO - 127.0.0.1:65009-[user] USER 'user' logged in.
2025-08-17 01:49:08,143 - pyftpdlib - INFO - 127.0.0.1:65009-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=52428832 seconds=28.735
2025-08-17 01:49:08,535 - RouterManager - INFO - Received chunk 1/1 for 50M (folder: file_002, target: node2)
2025-08-17 01:49:08,566 - RouterManager - INFO - Stored 50M in folder file_002 for node2
2025-08-17 01:49:08,629 - RouterManager - INFO - Target node node2 is active, forwarding file 50M
2025-08-17 01:49:09,054 - pyftpdlib - INFO - 127.0.0.1:65009-[user] FTP session closed (disconnect).
2025-08-17 01:51:13,363 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=22408 <<<
2025-08-17 01:51:13,363 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 01:51:13,364 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 01:51:13,365 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 01:51:23,790 - pyftpdlib - INFO - concurrency model: async
2025-08-17 01:51:23,791 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 01:51:23,799 - pyftpdlib - INFO - passive ports: None
2025-08-17 01:51:23,831 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=25124 <<<
2025-08-17 01:51:26,509 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 01:51:30,539 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:52:22,282 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 01:52:36,550 - pyftpdlib - INFO - 127.0.0.1:59087-[] FTP session opened (connect)
2025-08-17 01:52:36,645 - pyftpdlib - INFO - 127.0.0.1:59087-[user] USER 'user' logged in.
2025-08-17 01:52:37,114 - pyftpdlib - INFO - 127.0.0.1:59087-[user] FTP session closed (disconnect).
2025-08-17 01:52:37,160 - pyftpdlib - INFO - 127.0.0.1:59090-[] FTP session opened (connect)
2025-08-17 01:52:37,253 - pyftpdlib - INFO - 127.0.0.1:59090-[user] USER 'user' logged in.
2025-08-17 01:53:28,700 - pyftpdlib - INFO - 127.0.0.1:59090-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=104857632 seconds=50.859
2025-08-17 01:53:29,228 - RouterManager - INFO - Received chunk 1/2 for 200M (folder: file_003, target: node2)
2025-08-17 01:54:21,423 - pyftpdlib - INFO - 127.0.0.1:59090-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=104857632 seconds=51.328
2025-08-17 01:54:21,781 - RouterManager - INFO - Received chunk 2/2 for 200M (folder: file_003, target: node2)
2025-08-17 01:54:21,813 - RouterManager - INFO - Stored 200M in folder file_003 for node2
2025-08-17 01:54:21,859 - RouterManager - INFO - Target node node2 is active, forwarding file 200M
2025-08-17 01:54:22,341 - pyftpdlib - INFO - 127.0.0.1:59090-[user] FTP session closed (disconnect).
2025-08-17 02:00:47,589 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=25124 <<<
2025-08-17 02:00:47,589 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 02:00:47,590 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 02:00:47,591 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 02:02:22,275 - pyftpdlib - INFO - concurrency model: async
2025-08-17 02:02:22,276 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 02:02:22,277 - pyftpdlib - INFO - passive ports: None
2025-08-17 02:02:22,278 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=21328 <<<
2025-08-17 02:02:30,936 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 02:02:34,860 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 02:02:58,493 - pyftpdlib - INFO - 127.0.0.1:57332-[] FTP session opened (connect)
2025-08-17 02:02:58,587 - pyftpdlib - INFO - 127.0.0.1:57332-[user] USER 'user' logged in.
2025-08-17 02:02:59,087 - pyftpdlib - INFO - 127.0.0.1:57332-[user] FTP session closed (disconnect).
2025-08-17 02:02:59,119 - pyftpdlib - INFO - 127.0.0.1:57335-[] FTP session opened (connect)
2025-08-17 02:02:59,227 - pyftpdlib - INFO - 127.0.0.1:57335-[user] USER 'user' logged in.
2025-08-17 02:03:00,332 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.734
2025-08-17 02:03:00,645 - RouterManager - INFO - Received chunk 1/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:01,585 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.453
2025-08-17 02:03:01,738 - RouterManager - INFO - Received chunk 2/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:02,685 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.735
2025-08-17 02:03:02,824 - RouterManager - INFO - Received chunk 3/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:03,998 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.797
2025-08-17 02:03:04,168 - RouterManager - INFO - Received chunk 4/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:05,294 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.734
2025-08-17 02:03:05,479 - RouterManager - INFO - Received chunk 5/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:06,524 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.672
2025-08-17 02:03:06,682 - RouterManager - INFO - Received chunk 6/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:07,803 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.719
2025-08-17 02:03:07,989 - RouterManager - INFO - Received chunk 7/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:09,079 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.734
2025-08-17 02:03:09,235 - RouterManager - INFO - Received chunk 8/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:10,349 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.735
2025-08-17 02:03:10,537 - RouterManager - INFO - Received chunk 9/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:11,614 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.687
2025-08-17 02:03:11,752 - RouterManager - INFO - Received chunk 10/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:12,860 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.703
2025-08-17 02:03:13,030 - RouterManager - INFO - Received chunk 11/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:13,979 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.562
2025-08-17 02:03:14,166 - RouterManager - INFO - Received chunk 12/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:14,698 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.125
2025-08-17 02:03:14,836 - RouterManager - INFO - Received chunk 13/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:16,017 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.782
2025-08-17 02:03:16,190 - RouterManager - INFO - Received chunk 14/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:17,142 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.86
2025-08-17 02:03:17,314 - RouterManager - INFO - Received chunk 15/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:17,647 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.016
2025-08-17 02:03:17,769 - RouterManager - INFO - Received chunk 16/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:18,065 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.032
2025-08-17 02:03:18,081 - RouterManager - INFO - Received chunk 17/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:18,372 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.234
2025-08-17 02:03:18,479 - RouterManager - INFO - Received chunk 18/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:19,195 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.313
2025-08-17 02:03:19,300 - RouterManager - INFO - Received chunk 19/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:20,334 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.609
2025-08-17 02:03:20,523 - RouterManager - INFO - Received chunk 20/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:21,238 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.64
2025-08-17 02:03:21,316 - RouterManager - INFO - Received chunk 21/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:22,495 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.781
2025-08-17 02:03:22,665 - RouterManager - INFO - Received chunk 22/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:23,584 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.562
2025-08-17 02:03:23,787 - RouterManager - INFO - Received chunk 23/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:24,782 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.656
2025-08-17 02:03:24,956 - RouterManager - INFO - Received chunk 24/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:26,061 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.704
2025-08-17 02:03:26,247 - RouterManager - INFO - Received chunk 25/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:27,427 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.75
2025-08-17 02:03:27,630 - RouterManager - INFO - Received chunk 26/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:28,736 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.671
2025-08-17 02:03:28,953 - RouterManager - INFO - Received chunk 27/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:29,653 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.343
2025-08-17 02:03:29,777 - RouterManager - INFO - Received chunk 28/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:30,550 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.391
2025-08-17 02:03:30,753 - RouterManager - INFO - Received chunk 29/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:31,839 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.703
2025-08-17 02:03:31,978 - RouterManager - INFO - Received chunk 30/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:33,086 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.735
2025-08-17 02:03:33,272 - RouterManager - INFO - Received chunk 31/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:34,317 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.672
2025-08-17 02:03:34,503 - RouterManager - INFO - Received chunk 32/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:34,617 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.078
2025-08-17 02:03:34,769 - RouterManager - INFO - Received chunk 33/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:35,938 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.782
2025-08-17 02:03:36,125 - RouterManager - INFO - Received chunk 34/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:37,217 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.688
2025-08-17 02:03:37,438 - RouterManager - INFO - Received chunk 35/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:38,528 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.625
2025-08-17 02:03:38,731 - RouterManager - INFO - Received chunk 36/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:39,792 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.703
2025-08-17 02:03:39,993 - RouterManager - INFO - Received chunk 37/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:41,059 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.672
2025-08-17 02:03:41,217 - RouterManager - INFO - Received chunk 38/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:42,262 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.688
2025-08-17 02:03:42,419 - RouterManager - INFO - Received chunk 39/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:43,617 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.797
2025-08-17 02:03:43,836 - RouterManager - INFO - Received chunk 40/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:44,833 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.625
2025-08-17 02:03:45,003 - RouterManager - INFO - Received chunk 41/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:46,112 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.671
2025-08-17 02:03:46,284 - RouterManager - INFO - Received chunk 42/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:47,327 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.672
2025-08-17 02:03:47,529 - RouterManager - INFO - Received chunk 43/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:48,528 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.593
2025-08-17 02:03:48,732 - RouterManager - INFO - Received chunk 44/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:49,871 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.75
2025-08-17 02:03:50,089 - RouterManager - INFO - Received chunk 45/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:51,178 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.734
2025-08-17 02:03:51,366 - RouterManager - INFO - Received chunk 46/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:52,407 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.812
2025-08-17 02:03:52,611 - RouterManager - INFO - Received chunk 47/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:53,748 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.75
2025-08-17 02:03:53,949 - RouterManager - INFO - Received chunk 48/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:54,995 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.687
2025-08-17 02:03:55,196 - RouterManager - INFO - Received chunk 49/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:56,303 - pyftpdlib - INFO - 127.0.0.1:57335-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=1048608 seconds=0.688
2025-08-17 02:03:56,524 - RouterManager - INFO - Received chunk 50/50 for 50M (folder: file_003, target: node2)
2025-08-17 02:03:56,555 - RouterManager - INFO - Stored 50M in folder file_003 for node2
2025-08-17 02:03:56,603 - RouterManager - INFO - Target node node2 is active, forwarding file 50M
2025-08-17 02:03:56,775 - pyftpdlib - INFO - 127.0.0.1:57335-[user] FTP session closed (disconnect).
2025-08-17 02:07:52,143 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=21328 <<<
2025-08-17 02:07:52,144 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 02:07:52,144 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 02:07:52,145 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 02:07:56,030 - pyftpdlib - INFO - concurrency model: async
2025-08-17 02:07:56,031 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 02:07:56,031 - pyftpdlib - INFO - passive ports: None
2025-08-17 02:07:56,031 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=7524 <<<
2025-08-17 02:08:00,354 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 02:08:04,291 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 02:08:19,745 - pyftpdlib - INFO - 127.0.0.1:57789-[] FTP session opened (connect)
2025-08-17 02:08:19,886 - pyftpdlib - INFO - 127.0.0.1:57789-[user] USER 'user' logged in.
2025-08-17 02:08:20,445 - pyftpdlib - INFO - 127.0.0.1:57789-[user] FTP session closed (disconnect).
2025-08-17 02:08:20,477 - pyftpdlib - INFO - 127.0.0.1:57792-[] FTP session opened (connect)
2025-08-17 02:08:20,618 - pyftpdlib - INFO - 127.0.0.1:57792-[user] USER 'user' logged in.
2025-08-17 02:08:27,469 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.438
2025-08-17 02:08:27,750 - RouterManager - INFO - Received chunk 1/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:08:32,566 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=4.375
2025-08-17 02:08:32,738 - RouterManager - INFO - Received chunk 2/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:08:39,398 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.282
2025-08-17 02:08:39,603 - RouterManager - INFO - Received chunk 3/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:08:46,114 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.031
2025-08-17 02:08:46,302 - RouterManager - INFO - Received chunk 4/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:08:51,880 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=5.156
2025-08-17 02:08:52,061 - RouterManager - INFO - Received chunk 5/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:08:58,668 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.172
2025-08-17 02:08:58,856 - RouterManager - INFO - Received chunk 6/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:05,077 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=5.765
2025-08-17 02:09:05,249 - RouterManager - INFO - Received chunk 7/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:12,046 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.359
2025-08-17 02:09:12,250 - RouterManager - INFO - Received chunk 8/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:17,152 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=4.468
2025-08-17 02:09:17,356 - RouterManager - INFO - Received chunk 9/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:23,821 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.015
2025-08-17 02:09:23,992 - RouterManager - INFO - Received chunk 10/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:30,636 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.235
2025-08-17 02:09:30,823 - RouterManager - INFO - Received chunk 11/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:37,399 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.172
2025-08-17 02:09:37,603 - RouterManager - INFO - Received chunk 12/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:44,265 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.266
2025-08-17 02:09:44,419 - RouterManager - INFO - Received chunk 13/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:51,555 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.657
2025-08-17 02:09:51,774 - RouterManager - INFO - Received chunk 14/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:09:58,351 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.11
2025-08-17 02:09:58,551 - RouterManager - INFO - Received chunk 15/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:10:04,100 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=5.141
2025-08-17 02:10:04,299 - RouterManager - INFO - Received chunk 16/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:10:10,565 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=5.844
2025-08-17 02:10:10,812 - RouterManager - INFO - Received chunk 17/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:10:17,720 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.438
2025-08-17 02:10:17,909 - RouterManager - INFO - Received chunk 18/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:10:24,718 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.391
2025-08-17 02:10:24,906 - RouterManager - INFO - Received chunk 19/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:10:31,657 - pyftpdlib - INFO - 127.0.0.1:57792-[user] STOR E:\cloud-architecture-simulation\assets\server\200M completed=1 bytes=10485792 seconds=6.281
2025-08-17 02:10:31,890 - RouterManager - INFO - Received chunk 20/20 for 200M (folder: file_003, target: node2)
2025-08-17 02:10:31,936 - RouterManager - INFO - Stored 200M in folder file_003 for node2
2025-08-17 02:10:31,967 - RouterManager - INFO - Target node node2 is active, forwarding file 200M
2025-08-17 02:10:32,138 - pyftpdlib - INFO - 127.0.0.1:57792-[user] FTP session closed (disconnect).
2025-08-17 02:21:59,119 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=7524 <<<
2025-08-17 02:21:59,120 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 02:21:59,121 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 02:21:59,121 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 73, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 15:07:52,696 - pyftpdlib - INFO - concurrency model: async
2025-08-17 15:07:52,696 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 15:07:52,696 - pyftpdlib - INFO - passive ports: None
2025-08-17 15:07:52,696 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=25316 <<<
2025-08-17 15:08:07,092 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 15:08:22,659 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 15:08:54,065 - pyftpdlib - INFO - 127.0.0.1:53204-[] FTP session opened (connect)
2025-08-17 15:08:54,197 - pyftpdlib - INFO - 127.0.0.1:53204-[user] USER 'user' logged in.
2025-08-17 15:08:54,685 - pyftpdlib - INFO - 127.0.0.1:53204-[user] FTP session closed (disconnect).
2025-08-17 15:08:54,737 - pyftpdlib - INFO - 127.0.0.1:53207-[] FTP session opened (connect)
2025-08-17 15:08:54,816 - pyftpdlib - INFO - 127.0.0.1:53207-[user] USER 'user' logged in.
2025-08-17 15:09:01,285 - pyftpdlib - INFO - 127.0.0.1:53207-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=6.14
2025-08-17 15:09:01,397 - RouterManager - ERROR - Unknown sender IP 127.0.0.1
2025-08-17 15:09:01,514 - pyftpdlib - INFO - 127.0.0.1:53207-[user] FTP session closed (disconnect).
2025-08-17 15:11:43,816 - pyftpdlib - INFO - 127.0.0.1:53307-[] FTP session opened (connect)
2025-08-17 15:11:43,858 - pyftpdlib - INFO - 127.0.0.1:53307-[user] USER 'user' logged in.
2025-08-17 15:11:44,402 - pyftpdlib - INFO - 127.0.0.1:53307-[user] FTP session closed (disconnect).
2025-08-17 15:11:44,449 - pyftpdlib - INFO - 127.0.0.1:53310-[] FTP session opened (connect)
2025-08-17 15:11:44,525 - pyftpdlib - INFO - 127.0.0.1:53310-[user] USER 'user' logged in.
2025-08-17 15:11:51,221 - pyftpdlib - INFO - 127.0.0.1:53310-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=6.281
2025-08-17 15:11:51,356 - RouterManager - ERROR - Unknown sender IP 127.0.0.1
2025-08-17 15:11:51,439 - pyftpdlib - INFO - 127.0.0.1:53310-[user] FTP session closed (disconnect).
2025-08-17 15:16:52,826 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=25316 <<<
2025-08-17 15:16:52,826 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 15:16:52,826 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 15:16:52,826 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 75, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 15:17:46,524 - pyftpdlib - INFO - concurrency model: async
2025-08-17 15:17:46,524 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 15:17:46,524 - pyftpdlib - INFO - passive ports: None
2025-08-17 15:17:46,524 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=1616 <<<
2025-08-17 15:17:50,458 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 15:17:53,301 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 15:18:23,394 - pyftpdlib - INFO - 127.0.0.1:53454-[] FTP session opened (connect)
2025-08-17 15:18:23,506 - pyftpdlib - INFO - 127.0.0.1:53454-[user] USER 'user' logged in.
2025-08-17 15:18:23,985 - pyftpdlib - INFO - 127.0.0.1:53454-[user] FTP session closed (disconnect).
2025-08-17 15:18:24,023 - pyftpdlib - INFO - 127.0.0.1:53457-[] FTP session opened (connect)
2025-08-17 15:18:24,073 - pyftpdlib - INFO - 127.0.0.1:53457-[user] USER 'user' logged in.
2025-08-17 15:18:29,386 - pyftpdlib - INFO - 127.0.0.1:53457-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=4.922
2025-08-17 15:18:29,665 - RouterManager - INFO - Received chunk 1/5 for 50M (folder: file_003, from: node1, to: node2)
2025-08-17 15:18:35,876 - pyftpdlib - INFO - 127.0.0.1:53457-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=5.766
2025-08-17 15:18:36,035 - RouterManager - INFO - Received chunk 2/5 for 50M (folder: file_003, from: node1, to: node2)
2025-08-17 15:18:42,680 - pyftpdlib - INFO - 127.0.0.1:53457-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=6.204
2025-08-17 15:18:42,865 - RouterManager - INFO - Received chunk 3/5 for 50M (folder: file_003, from: node1, to: node2)
2025-08-17 15:18:48,980 - pyftpdlib - INFO - 127.0.0.1:53457-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=5.687
2025-08-17 15:18:49,163 - RouterManager - INFO - Received chunk 4/5 for 50M (folder: file_003, from: node1, to: node2)
2025-08-17 15:18:54,090 - pyftpdlib - INFO - 127.0.0.1:53457-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=4.485
2025-08-17 15:18:54,230 - RouterManager - INFO - Received chunk 5/5 for 50M (folder: file_003, from: node1, to: node2)
2025-08-17 15:18:54,253 - RouterManager - INFO - Stored 50M in folder file_003 for node2
2025-08-17 15:18:54,296 - RouterManager - INFO - Target node node2 is active, forwarding file 50M
2025-08-17 15:18:54,413 - pyftpdlib - INFO - 127.0.0.1:53457-[user] FTP session closed (disconnect).
2025-08-17 15:22:10,246 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 15:22:10,246 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=1616 <<<
2025-08-17 15:22:10,246 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 15:22:10,246 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 75, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 15:22:12,160 - pyftpdlib - INFO - concurrency model: async
2025-08-17 15:22:12,160 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 15:22:12,160 - pyftpdlib - INFO - passive ports: None
2025-08-17 15:22:12,160 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=8540 <<<
2025-08-17 15:22:15,319 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 15:22:19,012 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 15:22:31,907 - pyftpdlib - INFO - 127.0.0.1:53660-[] FTP session opened (connect)
2025-08-17 15:22:31,957 - pyftpdlib - INFO - 127.0.0.1:53660-[user] USER 'user' logged in.
2025-08-17 15:22:32,455 - pyftpdlib - INFO - 127.0.0.1:53660-[user] FTP session closed (disconnect).
2025-08-17 15:22:32,496 - pyftpdlib - INFO - 127.0.0.1:53663-[] FTP session opened (connect)
2025-08-17 15:22:32,556 - pyftpdlib - INFO - 127.0.0.1:53663-[user] USER 'user' logged in.
2025-08-17 15:22:38,831 - pyftpdlib - INFO - 127.0.0.1:53663-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=5.89
2025-08-17 15:22:38,921 - RouterManager - ERROR - No link exists between sender node1 and target node2
2025-08-17 15:22:39,012 - pyftpdlib - INFO - 127.0.0.1:53663-[user] FTP session closed (disconnect).
2025-08-17 15:23:34,385 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=8540 <<<
2025-08-17 15:23:34,385 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 15:23:34,385 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 15:23:34,385 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 75, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 15:32:23,590 - pyftpdlib - INFO - concurrency model: async
2025-08-17 15:32:23,590 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 15:32:23,590 - pyftpdlib - INFO - passive ports: None
2025-08-17 15:32:23,590 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=13540 <<<
2025-08-17 15:32:27,880 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 15:32:34,876 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 15:33:21,738 - pyftpdlib - INFO - 127.0.0.1:53892-[] FTP session opened (connect)
2025-08-17 15:33:21,820 - pyftpdlib - INFO - 127.0.0.1:53892-[user] USER 'user' logged in.
2025-08-17 15:33:22,404 - pyftpdlib - INFO - 127.0.0.1:53892-[user] FTP session closed (disconnect).
2025-08-17 15:33:22,462 - pyftpdlib - INFO - 127.0.0.1:53895-[] FTP session opened (connect)
2025-08-17 15:33:22,538 - pyftpdlib - INFO - 127.0.0.1:53895-[user] USER 'user' logged in.
2025-08-17 15:33:29,281 - pyftpdlib - INFO - 127.0.0.1:53895-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=6.343
2025-08-17 15:33:29,367 - RouterManager - ERROR - No link exists between sender node1 and target node2
2025-08-17 15:33:29,434 - pyftpdlib - INFO - 127.0.0.1:53895-[user] FTP session closed (disconnect).
2025-08-17 15:33:36,048 - pyftpdlib - INFO - 127.0.0.1:53902-[] FTP session opened (connect)
2025-08-17 15:33:36,116 - pyftpdlib - INFO - 127.0.0.1:53902-[user] USER 'user' logged in.
2025-08-17 15:33:36,646 - pyftpdlib - INFO - 127.0.0.1:53902-[user] FTP session closed (disconnect).
2025-08-17 15:33:36,715 - pyftpdlib - INFO - 127.0.0.1:53905-[] FTP session opened (connect)
2025-08-17 15:33:36,832 - pyftpdlib - INFO - 127.0.0.1:53905-[user] USER 'user' logged in.
2025-08-17 15:33:42,970 - pyftpdlib - INFO - 127.0.0.1:53905-[user] STOR E:\cloud-architecture-simulation\assets\server\50M completed=1 bytes=10485792 seconds=5.766
2025-08-17 15:33:43,080 - RouterManager - ERROR - No link exists between sender node1 and target node2
2025-08-17 15:33:43,180 - pyftpdlib - INFO - 127.0.0.1:53905-[user] FTP session closed (disconnect).
2025-08-17 15:38:33,741 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=13540 <<<
2025-08-17 15:38:33,741 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 15:38:33,741 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 15:38:33,741 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "E:\cloud-architecture-simulation\router_manager.py", line 75, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 16:22:21,320 - pyftpdlib - INFO - concurrency model: async
2025-08-17 16:22:21,320 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 16:22:21,320 - pyftpdlib - INFO - passive ports: None
2025-08-17 16:22:21,320 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=18488 <<<
2025-08-17 16:22:37,279 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 16:22:51,455 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 16:23:17,475 - pyftpdlib - INFO - 127.0.0.1:56511-[] FTP session opened (connect)
2025-08-17 16:23:17,574 - pyftpdlib - INFO - 127.0.0.1:56511-[user] USER 'user' logged in.
2025-08-17 16:23:18,102 - pyftpdlib - INFO - 127.0.0.1:56511-[user] FTP session closed (disconnect).
2025-08-17 16:23:18,134 - pyftpdlib - INFO - 127.0.0.1:56514-[] FTP session opened (connect)
2025-08-17 16:23:18,273 - pyftpdlib - INFO - 127.0.0.1:56514-[user] USER 'user' logged in.
2025-08-17 16:23:23,635 - pyftpdlib - INFO - 127.0.0.1:56514-[user] STOR C:\Users\JUAN MIKE\Desktop\Projets\cloud-architecture-simulation\assets\server\10M completed=1 bytes=10485792 seconds=4.985
2025-08-17 16:23:23,776 - RouterManager - ERROR - No link exists between sender node1 and target node2
2025-08-17 16:23:23,859 - pyftpdlib - INFO - 127.0.0.1:56514-[user] FTP session closed (disconnect).
2025-08-17 16:26:08,372 - pyftpdlib - INFO - 127.0.0.1:56554-[] FTP session opened (connect)
2025-08-17 16:26:08,468 - pyftpdlib - INFO - 127.0.0.1:56554-[user] USER 'user' logged in.
2025-08-17 16:26:08,952 - pyftpdlib - INFO - 127.0.0.1:56554-[user] FTP session closed (disconnect).
2025-08-17 16:26:08,973 - pyftpdlib - INFO - 127.0.0.1:56557-[] FTP session opened (connect)
2025-08-17 16:26:09,101 - pyftpdlib - INFO - 127.0.0.1:56557-[user] USER 'user' logged in.
2025-08-17 16:26:14,988 - pyftpdlib - INFO - 127.0.0.1:56557-[user] STOR C:\Users\JUAN MIKE\Desktop\Projets\cloud-architecture-simulation\assets\server\10M completed=1 bytes=10485792 seconds=5.562
2025-08-17 16:26:15,360 - RouterManager - INFO - Received chunk 1/1 for 10M (folder: file_001, from: node1, to: node2)
2025-08-17 16:26:15,384 - RouterManager - INFO - Stored 10M in folder file_001 for node2
2025-08-17 16:26:15,415 - RouterManager - INFO - Target node node2 is active, forwarding file 10M
2025-08-17 16:26:15,567 - pyftpdlib - INFO - 127.0.0.1:56557-[user] FTP session closed (disconnect).
2025-08-17 17:56:08,836 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 17:56:08,836 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=18488 <<<
2025-08-17 17:56:08,836 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 17:56:08,836 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "C:\Users\JUAN MIKE\Desktop\Projets\cloud-architecture-simulation\router_manager.py", line 75, in _handle_socket_connections
    threading.Thread(target=self._process_socket_message, args=(client_socket,), daemon=True).start()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 17:57:16,508 - pyftpdlib - INFO - concurrency model: async
2025-08-17 17:57:16,508 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 17:57:16,508 - pyftpdlib - INFO - passive ports: None
2025-08-17 17:57:16,508 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=23716 <<<
2025-08-17 17:58:22,053 - RouterManager - INFO - Node cloud_node1 started, checking for pending files
2025-08-17 17:59:24,998 - RouterManager - INFO - Node cloud_node2 started, checking for pending files
2025-08-17 17:59:53,827 - RouterManager - INFO - Node cloud_node3 started, checking for pending files
2025-08-17 18:00:03,446 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 18:00:16,781 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 18:00:54,256 - pyftpdlib - INFO - 127.0.0.1:50751-[] FTP session opened (connect)
2025-08-17 18:00:54,353 - pyftpdlib - INFO - 127.0.0.1:50751-[user] USER 'user' logged in.
2025-08-17 18:00:54,913 - pyftpdlib - INFO - 127.0.0.1:50751-[user] FTP session closed (disconnect).
2025-08-17 18:00:54,959 - pyftpdlib - INFO - 127.0.0.1:50754-[] FTP session opened (connect)
2025-08-17 18:00:55,019 - pyftpdlib - INFO - 127.0.0.1:50754-[user] USER 'user' logged in.
2025-08-17 18:01:01,899 - pyftpdlib - INFO - 127.0.0.1:50754-[user] STOR C:\Users\JUAN MIKE\Desktop\Projets\cloud-architecture-simulation\assets\server\10M completed=1 bytes=10485792 seconds=6.453
2025-08-17 18:01:02,115 - RouterManager - INFO - Received chunk 1/1 for 10M (folder: file_001, target: node2, sender: node1)
2025-08-17 18:01:02,152 - RouterManager - INFO - Stored 10M in folder file_001 for node2 from node1
2025-08-17 18:01:02,175 - RouterManager - INFO - Target node node2 is active, forwarding file 10M
2025-08-17 18:01:02,216 - RouterManager - INFO - Cloud node cloud_node1 is active, forwarding file 10M from node1
2025-08-17 18:01:02,969 - RouterManager - INFO - Cloud node cloud_node2 is active, forwarding file 10M from node1
2025-08-17 18:01:03,577 - RouterManager - INFO - Cloud node cloud_node3 is active, forwarding file 10M from node1
2025-08-17 18:01:04,433 - pyftpdlib - INFO - 127.0.0.1:50754-[user] FTP session closed (disconnect).
2025-08-17 18:13:47,940 - RouterManager - INFO - FTP server stopped for 127.0.0.1
2025-08-17 18:13:47,940 - pyftpdlib - INFO - >>> shutting down FTP server, 0 socket(s), pid=23716 <<<
2025-08-17 18:13:47,940 - RouterManager - INFO - Socket server stopped for 127.0.0.1
2025-08-17 18:13:47,940 - RouterManager - ERROR - Socket server error: [WinError 10038] An operation was attempted on something that is not a socket
Traceback (most recent call last):
  File "C:\Users\JUAN MIKE\Desktop\Projets\cloud-architecture-simulation\router_manager.py", line 74, in _handle_socket_connections
    client_socket, addr = self.socket_server.accept()
  File "C:\Users\JUAN MIKE\.conda\envs\Cloudsim\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
OSError: [WinError 10038] An operation was attempted on something that is not a socket
2025-08-17 18:14:17,174 - pyftpdlib - INFO - concurrency model: async
2025-08-17 18:14:17,174 - pyftpdlib - INFO - masquerade (NAT) address: None
2025-08-17 18:14:17,174 - pyftpdlib - INFO - passive ports: None
2025-08-17 18:14:17,174 - pyftpdlib - INFO - >>> starting FTP server on 0.0.0.0:2120, pid=13256 <<<
2025-08-17 18:14:46,580 - RouterManager - INFO - Node cloud_node1 started, checking for pending files
2025-08-17 18:14:51,034 - RouterManager - INFO - Node cloud_node2 started, checking for pending files
2025-08-17 18:14:54,604 - RouterManager - INFO - Node cloud_node3 started, checking for pending files
2025-08-17 18:15:24,365 - RouterManager - INFO - Node node1 started, checking for pending files
2025-08-17 18:15:29,137 - RouterManager - INFO - Node node2 started, checking for pending files
2025-08-17 18:16:28,609 - pyftpdlib - INFO - 127.0.0.1:50981-[] FTP session opened (connect)
2025-08-17 18:16:28,703 - pyftpdlib - INFO - 127.0.0.1:50981-[user] USER 'user' logged in.
2025-08-17 18:16:29,238 - pyftpdlib - INFO - 127.0.0.1:50981-[user] FTP session closed (disconnect).
2025-08-17 18:16:29,287 - pyftpdlib - INFO - 127.0.0.1:50984-[] FTP session opened (connect)
2025-08-17 18:16:29,386 - pyftpdlib - INFO - 127.0.0.1:50984-[user] USER 'user' logged in.
2025-08-17 18:16:36,101 - pyftpdlib - INFO - 127.0.0.1:50984-[user] STOR C:\Users\JUAN MIKE\Desktop\Projets\cloud-architecture-simulation\assets\server\10M completed=1 bytes=10485792 seconds=6.281
2025-08-17 18:16:36,363 - RouterManager - INFO - Received chunk 1/1 for 10M (folder: file_001, target: node2, sender: node1)
2025-08-17 18:16:36,401 - RouterManager - INFO - Stored 10M in folder file_001 for node2 from node1
2025-08-17 18:16:36,434 - RouterManager - INFO - Target node node2 is active, forwarding file 10M
2025-08-17 18:16:36,473 - RouterManager - INFO - Cloud node cloud_node1 is active, forwarding file 10M from node1
2025-08-17 18:16:37,080 - RouterManager - INFO - Cloud node cloud_node2 is active, forwarding file 10M from node1
2025-08-17 18:16:37,529 - RouterManager - INFO - Cloud node cloud_node3 is active, forwarding file 10M from node1
2025-08-17 18:16:38,471 - pyftpdlib - INFO - 127.0.0.1:50984-[user] FTP session closed (disconnect).
